2021-01-18_Liu_reactions.pbtxt

```diff
@@ -0,0 +1,1480 @@
+    }
+  }
+  inputs {
+    key: "dichloromethane"
+    value {
+      components {
+        identifiers {
+          type: SMILES
+          value: "C(Cl)Cl"
+        }
+        identifiers {
+          type: NAME
+          value: "dichloromethane"
+        }
+        amount {
+          volume {
+            value: 250.0
+            units: MILLILITER
+          }
+        }
+        reaction_role: SOLVENT
+        preparations {
+          type: DRIED
+          details: "purified by passage under argon pressure through two packed columns of neutral alumina and copper(II) oxide"
+        }
+        source {
+          vendor: "J. T. Baker"
+        }
+      }
+      addition_order: 3
+    }
+  }
+  inputs {
+    key: "pivaloyl chloride"
+    value {
+      components {
+        identifiers {
+          type: SMILES
+          value: "CC(C)(C)C(=O)Cl"
+        }
+        identifiers {
+          type: NAME
+          value: "pivaloyl chloride"
+        }
+        amount {
+          volume {
+            value: 12.9
+            units: MILLILITER
+          }
+        }
+        reaction_role: REACTANT
+        preparations {
+          type: NONE
+          details: "used as received"
+        }
+        source {
+          vendor: "Alfa Aesar"
+        }
+      }
+      addition_order: 4
+      addition_speed {
+        type: DROPWISE
+      }
+      addition_duration {
+        value: 5.0
+        units: MINUTE
+      }
+      addition_device {
+        type: SYRINGE
+        details: "plastic 30-mL syringe"
+      }
+    }
+  }
+  setup {
+    vessel {
+      type {
+        type: ROUND_BOTTOM_FLASK
+        details: "single-necked"
+      }
+      material {
+        type: GLASS
+      }
+      preparations {
+        type: OVEN_DRIED
+      }
+      preparations {
+        type: PURGED
+        details: "with nitrogen"
+      }
+      attachments {
+        type: SEPTUM
+        details: "rubber"
+      }
+      volume {
+        value: 500.0
+        units: MILLILITER
+      }
+    }
+    is_automated: false
+    environment {
+      type: FUME_HOOD
+    }
+  }
+  conditions {
+    temperature {
+      control {
+        type: ICE_BATH
+        details: "reaction started in ice bath before addition of pivaloyl \nchloride, then allowed to warm to 23 C, then stirred for additional 6 hours at \nRT (23 degC)"
+      }
+      setpoint {
+        value: 0.0
+        units: CELSIUS
+      }
+    }
+    pressure {
+      control {
+        type: SLIGHT_POSITIVE
+        details: "sealed with rubber septum, pierced with needle connected to manifold"
+      }
+      atmosphere {
+        type: NITROGEN
+        details: "dry nitrogen"
+      }
+    }
+    stirring {
+      method {
+        type: STIR_BAR
+      }
+    }
+    conditions_are_dynamic: true
+    details: "reaction started in ice bath before addition of\npivaloyl chloride, then allowed to warm to 23 C, then stirred for \'\nan additional 6 hours at RT (23 degC)"
```

Comment:
Looks like there is a stray quotation mark here. The newline symbols `\n` aren't ideal for these long, multi-line strings.